man/chage.1.xml

```diff
@@ -75,12 +77,19 @@
 	</term>
 	<listitem>
 	  <para>
-	    Set the number of days since 1970-01-01 when the password
-	    was last changed. The date may also be expressed in the format
-	    YYYY-MM-DD.
-	    If the <replaceable>LAST_DAY</replaceable> is set to
-	    <emphasis>0</emphasis> the user is forced to change his password
-	    on the next log on.
+	    Set the number of days since 1970-01-01
+	    when the password was last changed.
+	    The value can be specified as a date in the YYYY-MM-DD format
+	    or as a number of days since 1970-01-01.
```

Comment:
The wording here is inconsistent with the proposed wording for -E.  (See my other comment in -E.)